kernel/public/test-scenes/0.101.move-inside-scene/game.ts

```diff
@@ -0,0 +1,13 @@
+import { BoxShape, engine, Entity, log, moveTo, OnPointerUp, Transform, Vector3 } from 'decentraland-ecs/src'
+
+const e = new Entity()
+e.addComponent(new BoxShape())
+e.addComponent(new Transform({ position: new Vector3(4, 0, 6) }))
+e.addComponent(
+  new OnPointerUp((e) => {
```

Comment:
Just tried the scene locally, and it works great!!
Super minor change to make the scene more user friendly:
We could change `OnPointerUp` for `OnPointerDown` and add a hover text that says something like "Move player"
You can see here how to set the hover text https://docs.decentraland.org/development-guide/click-events/